flink-core/src/test/java/org/apache/flink/api/common/ExecutionConfigTest.java

```diff
@@ -159,4 +160,21 @@ public void testExecutionConfigSerialization() throws IOException, ClassNotFound
 		assertEquals(sysoutLoggingEnabled, copy1.isSysoutLoggingEnabled());
 		assertEquals(parallelism, copy1.getParallelism());
 	}
+
+	@Test
+	public void testGlobalParameters() {
```

Comment:
I think it would be enough to test that when we call `ExecutionConfig#getGlobalJobParameters()` without setting the value before, returns a non null value. In general, we should not test multiple things in the same test.